logging.js

```diff
@@ -157,17 +157,22 @@ export class LogBuilder {
 }
 
 export class LoggingClient {
-	constructor(appId, logger) {
+	constructor(appId, shouldThrottle, logger) {
```

Comment:
Could also not pass it in the constructor, default it to `false` in the constructor and then provide a getter/setter. 🤷  Would avoid needing to change all the tests to pass in `false`.